apiserver/gui.go

```diff
@@ -290,22 +244,23 @@ func uncompressGUI(r io.Reader, sourceDir, targetDir string) error {
 
 // guiHandler serves the Juju GUI.
 type guiHandler struct {
-	baseModelURLPath string
-	baseGUIURLPath   string
-	rootDir          string
-	hash             string
-	uuid             string
+	ctxt     httpContext
+	basePath string
+	rootDir  string
+	hash     string
 }
 
 // serveStatic serves the GUI static files.
 func (h *guiHandler) serveStatic(w http.ResponseWriter, req *http.Request) {
+	logger.Debugf("serving Juju GUI static files")
```

Comment:
should this not print out the path it's serving as well? Also for the other new serve logs below